turbinia/processors/mount_local.py

```diff
@@ -48,17 +50,18 @@ def PreprocessLosetup(source_path):
 
   if not os.path.exists(source_path):
     raise TurbiniaException(
-        'Cannot process non-existing source_path {0!s}'.format(source_path))
+        'Cannot mount non-existing source_path {0!s}'.format(source_path))
```

Comment:
minor nit (sorry I missed this last time): We're not actually mounting it in this preprocessor.  Maybe 'Cannot create loopback for ..."?